semperfiwebdesign/all-in-one-seo-pack
# Add last change timestamp for homepage & tax terms

Issue #262

## Proposed changes

Adds the Last Change timestamp for the homepage and taxonomy terms.
This supports taxonomy terms with multiple attached post types.

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist


## Testing instructions

1. Check whether the Last Change timestamp for the homepage is based on the last modified Post.

2. Check whether the Last Change timestamp for taxonomies (Categories, Tags, custom taxonomies) is based on the last modified post of the post type that is assigned to it.

3. Check whether the Last Change timestamp for taxonomies with multiple attached post types* is based on the timestamp of the last modified post that is assigned to it, regardless of its type. 

_*e.g. use the Custom Post Type UI plugin to create a new custom taxonomy with Posts and Pages as its post types. Assign both a Post and Page to it; then update them randomly and check whether the timestamp is based on the one that was most recently updated._

You might also want to test this on a large test site to see if this doesn't impact performance too much, but it shouldn't based on my test results.
